### DISCUSSION TITLE
Updating an exiting library of albums and tracks.

### DISCUSSION THREAD

**Discussion by Arskin-Fertrubble** [beetbox/beets#4139] [Q&A] [answered]
ED: Typo in subject. Should be
"Updating an **existing** library of albums and tracks"
How do I update metadata and artwork for albums and tracks in an established beets library? Do I use IMPORT? if so, what switches? Or is there a plugin or other command?

NB: I found the documentation unclear about this, so thanks in advance.

**Comment by DjSlash** [comment, marked as answer]
Use the [mbsync](https://beets.readthedocs.io/en/stable/plugins/mbsync.html) plugin. It will update the metadata for your files. Regarding the artwork, the [fetchart](https://beets.readthedocs.io/en/stable/plugins/fetchart.html) plugin normally only adds artwork where it doesn't exist. But you can use the --force option to look up art where it does exist.
